Jenkinsfile-build

```diff
@@ -18,10 +21,14 @@ pipeline {
                   }
                 }
                 steps {
-                  checkout([$class: 'GitSCM', branches: [[name: '*/master']], userRemoteConfigs: [[url: 'https://github.com/UKHomeOffice/keycloak-theme-govuk.git']]])
-                  sh 'npm install'
-                  sh 'npm run build'
-                  stash includes: "govuk/**", name: "govuk"
+                  script {
```

Comment:
Minor thing, but I'd suggest wrapping the `script` section just around the library call, rather than around everything. We should [only have to use `script` in unusual situations](https://www.jenkins.io/doc/book/pipeline/syntax/#script) (and it's a bit annoying that this is one of them), so we should at least fence it off from the rest of the code which doesn't need it.

Reply:
Resolved